org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeNetworkStateSerializerImpl.java

```diff
@@ -8,6 +8,9 @@
  */
```

Comment:
I think this class is probably too complex now. It still contains the hacks we added to avoid concurrency, but that should not be required now. I'd suggest to use the new ZigBeeDataStore class from ZSS. I think you've partly used that, but also kept the concurrency hacks - do you think this is really needed? One of the main reasons to rewrite the class was to avoid this sort of thing.

Reply:
What my PR is doing is simply implementing the new interface, but still using the very same "dump" implementation as before, i.e. serializing all nodes instead of just the one which was updated, etc.

So are you suggesting to create a new implementation for your `ZigBeeNetworkDataStore` class?